packages/react-core/src/components/Card/Card.tsx

```diff
@@ -20,40 +22,63 @@ export interface CardProps extends React.HTMLProps<HTMLElement>, OUIAProps {
   isSelected?: boolean;
   /** Modifies the card to include flat styling */
   isFlat?: boolean;
+  /** Flag indicating if a card is expanded. Modifies the card to be expandable. */
+  isExpanded?: boolean;
 }
 
+interface CardContextProps {
+  cardId: string;
+  isExpanded: boolean;
+}
+
+export const CardContext = React.createContext<Partial<CardContextProps>>({
+  cardId: '',
+  isExpanded: false
+});
+
 export const Card: React.FunctionComponent<CardProps> = ({
   children = null,
+  id = '',
```

Comment:
```suggestion
  id,
```
just so empty div doesn't get rendered